chapters/annotations.tex

```diff
@@ -1747,13 +1747,12 @@ \subsubsection{Conversion rules}\label{conversion-rules}
    {"y0=0", "k1=0", "k2=0"}, {"y0=%y0%", "k1=%y0%*%k1%", "k2=%y0%*%k2%"},
    true);
 convertClass(
-   "Modelica.Electrical.Machines.BasicMachines.QuasiStationaryDCMachines",
-   "Modelica.Electrical.Machines.BasicMachines.QuasiStaticDCMachines");
-convertElement("Modelica.Electrical.Machines.Interfaces.PartialBasicDCMachine",
-               "quasiStationary", "quasiStatic");
-convertElement("Modelica.Electrical.Machines.BasicMachines." +
-                "QuasiStationaryDCMachines.DC_ElectricalExcited",
-               "quasiStationary", "quasiStatic");
+   "My.Library.BadPackage",
+   "My.Library.Package");
+convertElement("My.Library.BadPackage.PartialBase",
+               "bad", "correct");
+convertElement("My.Library.BadPackage.ActualClass",
+               "bad", "correct");
```

Comment:
Shouldn't these be indented to the same level as the `convertClass` above?

Reply:
I changed all of the new code to be consistently indented as previous code.

Reply:
Looks good now.  Resolving.